modules/search/instant-search/lib/sort.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { SORT_DIRECTION_ASC, SORT_DIRECTION_DESC } from './constants';
+
+export function getSortOptions() {
```

Comment:
Since this function is called in every `SearchSortWidget.render` call, it might be worth defining the returned object as a constant outside of the function.

As-is, this function creates a new object on every function call.